[12.0][ADD] sale_timesheet_limit_date (#890)
OCA/sale-workflow: hr_timesheet_limit_sale_date/readme/DESCRIPTION.rst

> This module adds a date field to sale order and invoice. This date is limitation

Reviewer: ```suggestion
This module adds a date field to sale order and invoice. This date limits the
```

selection of timesheet lines for including them in sale order and invoice.

> When create an invoice limit date will be inherited from sale order, then date

Reviewer: ```suggestion
When creating an invoice the limit date will be inherited from the sale order, then date
```
